app/views/layout.scala.html

```diff
@@ -78,6 +78,7 @@
 @additionalScripts = {
     <script src='@routes.Assets.versioned("jquery/jquery-3.6.0.min.js")'></script>
     <script src='@controllers.routes.Assets.versioned("jquery/jquery-ui.min.js")'></script>
+    <link rel="stylesheet" href='@routes.Assets.versioned("../assets/stylesheets/print.css")' type="text/css"/>
```

Comment:
Do we need the / at the end i.e. should be: 
<link rel="stylesheet" href='@routes.Assets.versioned("../assets/stylesheets/print.css")' type="text/css">

Reply:
Pushed up changes - resolved